scripts/mutation-test.sh

```diff
@@ -1,22 +1,35 @@
 #!/usr/bin/env bash
 
 set -eo pipefail
+oIFS="$IFS"; IFS=, ; set -- $1 ; IFS="$oIFS"
 
 DISABLED_MUTATORS='branch/*'
 
 # Only consider the following:
-# * go files in types or keeper packages
+# * go files in types, keeper, or module root directories
 # * ignore test and Protobuf files
-MUTATION_SOURCES=$(find ./x -type f \( -path '*/keeper/*' -or -path '*/types/*' \) \( -name '*.go' -and -not -name '*_test.go' -and -not -name '*pb*' \))
+MUTATION_SOURCES=$(find ./x -type f \( -path '*/keeper/*' -or -path '*/types/*' -or -path '*' \) \( -name '*.go' -and -not -name '*_test.go' -and -not -name '*pb*' \))
```

Comment:
So we are allowing any path according to `-or -path '*'`? Does it make sense to not apply any path filtering then?

Reply:
Oof I meant root directory files but I didn't think that included the folders themselves, let me look into this, good catch

Reply:
Great catch! Most recent commit should solve this problem by specifying a max depth after pulling what we want from keeper and types

Reply:
Why are we excluding paths deeper than 2? Is the goal to exclude files in the `client` folder?

Reply:
We want keeper folder and types folder which is the first cmd. Then the second command starts in the x folder, and two up would be the root module folder (because x counts as one) so this gives us any root folder files for the selected modules (I used prints to ensure this works as intended)

Reply:
That makes sense, why do we only want the root files of each module? What about pool-module packages and alike??

Reply:
Hmm, I didn't want to drift too far away from the original implementation since I lacked context on mutation testing. In fact I would have completely left it alone if twap had the same structure as the other modules. You might be right though, we should probably test this as well